tools/codegen/core/gen_upb_api.sh

```diff
@@ -36,6 +43,11 @@ proto_files=( \
   "google/protobuf/wrappers.proto" \
   "google/rpc/status.proto" \
   "gogoproto/gogo.proto" \
+  "grpc/health/v1/health.proto" \
+  "grpc/lb/v1/load_balancer.proto" \
+  "src/core/tsi/alts/handshaker/proto/handshaker.proto" \
```

Comment:
Maybe we should alphabetize this list. I didn't realize it wasn't.

Reply:
Done :)